react/CoolTable.js

```diff
@@ -0,0 +1,40 @@
+const React = require('react')
+const ReactDOM = require('react-dom')
+
+const MUIDataTable = require("mui-datatables").default;
+
+const columns = ["Name", "Company", "City", "State"];
+
+const data = [
+ ["Joe James", "Test Corp", "Yonkers", "NY"],
+ ["John Walsh", "Test Corp", "Hartford", "CT"],
+ ["Bob Herm", "Test Corp", "Tampa", "FL"],
+ ["James Houston", "Test Corp", "Dallas", "TX"],
+];
+
+const options = {
+  filterType: 'checkbox',
+};
+
+const CoolTable = function () {
```

Comment:
I know this is just an example, but should the columns and data be able to be passed here via props? Thinking we should probably not add this to the codebase unless it's a working component.

Reply:
Yep haha. I decided not to continue with that Component example. But I removed it from the codebase! And just to be clear, I'm not suggesting we add any of this to the codebase. My whole goal was a proof of concept. We can figure out what to do with all this stuff tomorrow after the demo!